### ISSUE TITLE
a11y_FormFieldContainer> Default> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#496]
**Describe the bug**
FormFieldContainer>> "1 violations" is appearing for "Default". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/formfieldcontainer--default'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for FormFieldContainer> Default>.

**Screenshots**
https://somup.com/c3n2j3ZWNb

**Desktop:**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by silvalaura** [comment]
FormFieldContainer is an internal component that has been removed from Storybook, do this issue is no longer relevant. Closing as won't fix.
